spec/controllers/author/submissions_controller_spec.rb

```diff
@@ -12,24 +12,75 @@
   describe '#new' do
     it 'initializes a new submission for an author' do
       expect(get: new_author_submission_path).to route_to(controller: 'author/submissions', action: 'new')
-      # get :new
-      # expect(response).to redirect_to('')
     end
   end
 
   describe '#create' do
     it 'creates a new submission for an author' do
-      # submission = FactoryBot.create :submission, :collecting_committee
       expect(post: author_submissions_path).to route_to(controller: 'author/submissions', action: 'create')
-      # get :post, params: { submission:  submission.attributes }
     end
   end
 
   describe '#edit' do
-    it 'creates a new submission for an author' do
-      submission = FactoryBot.create :submission
+    it 'routes to the edit page' do
+      submission = FactoryBot.create(:submission, acknowledgment_page_submitted_at: nil)
       expect(get: edit_author_submission_path(submission.id)).to route_to(controller: 'author/submissions', action: 'edit', id: submission.id.to_s)
     end
+
+    if current_partner.graduate?
+      it 'redirects to the acknowledge page if acknowledgement page has not been viewed' do
+        oidc_authorize_author
+        submission = FactoryBot.create(:submission, acknowledgment_page_submitted_at: nil)
+        params = { id: submission.id.to_s }
+        allow(controller).to receive(:find_submission).and_return(submission)
+        expect(get(:edit, params:)).to redirect_to author_submission_acknowledge_path(submission.id)
+      end
+    end
```

Comment:
I think it would be good to also test what happens when `acknowledgment_page_submitted_at` is set and when the partner is not graduate here.